src/humanPose/draw.js

```diff
@@ -1463,14 +439,17 @@ function setHumanPosesVisible(visible) {
  * @param {boolean} visible - whether to show or not
  */
 function setChildHumanPosesVisible(visible) {
```

Comment:
This feels like it should just affect the active HPA. Also, in UI it is a toggle in its own settings.